stacker/actions/base.py

```diff
@@ -121,9 +123,13 @@ def s3_stack_push(self, blueprint, force=False):
         return template_url
 
     def execute(self, *args, **kwargs):
-        self.pre_run(*args, **kwargs)
-        self.run(*args, **kwargs)
-        self.post_run(*args, **kwargs)
+        try:
+            self.pre_run(*args, **kwargs)
+            self.run(*args, **kwargs)
+            self.post_run(*args, **kwargs)
+        except PlanFailed as e:
+            logger.error(e.message)
```

Comment:
This would hide the traceback stack, right?  Should we instead do `logger.exception`?

Reply:
`PlainFailed` should only be raised when CF reaches a failure state, which should mean the traceback doesn't help much. Other kinds of exceptions should bubble through and end execution.